docs/building-apps/wallet/sep24.mdx

```diff
@@ -173,6 +241,19 @@ val watcher = sep24.watcher()
 val result = watcher.watchOneTransaction(token, "transaction id")
 ```
 
+```typescript
+let watcher = anchor.watcher();
+
+let result = watcher.watchOneTransaction({
+  authToken,
+  assetCode,
```

Comment:
Why do we need assetCode when watching one transaction?

Reply:
Ok figure it out later, as I don't think it should be necessary

Reply:
@CassioMG is more familiar with this code than I am. But from what I understand we're using the asset code as the key for the registry to keep track of transactions: https://github.com/stellar/typescript-wallet-sdk/blob/b0a1ff12c2531c9526ebc59bfbed285f1db04d5d/src/walletSdk/Watcher/index.ts#L100-L113